### PR TITLE
[CP stag] Reverts Android Keyboard Spacer changes

### PR DESCRIPTION
**Pull request by Luke9389**
### Details

This PR reverts this one: https://github.com/Expensify/App/pull/10162/files

### Fixed Issues

We think it may fix this: https://github.com/Expensify/App/issues/10520
and this: https://github.com/Expensify/App/issues/10539

### CODE REVIEW

File: src/components/KeyboardSpacer/index.ios.js [hunk 1]
```diff
@@ -3,20 +3,31 @@
  * keyboard allowing the user to see what they are typing.
  */
 import React from 'react';
```
Comment (Beamanator) [inline src/components/KeyboardSpacer/index.ios.js:5]:
```suggestion
import _ from 'underscore';
import React from 'react';
```

Comment (Beamanator) [inline src/components/KeyboardSpacer/index.ios.js:5]:
```suggestion
import _ from 'underscore';
import React from 'react';
```

Comment (Luke9389) [reply src/components/KeyboardSpacer/index.ios.js:6]: Thanks!

Comment (Luke9389) [reply src/components/KeyboardSpacer/index.ios.js:6]: I think this is a duplicate

Comment (Beamanator) [reply src/components/KeyboardSpacer/index.ios.js:6]: shhhhhhh :D

**Comment by roryabraham** [comment]
Was able to confirm that this _does not_ fix https://github.com/Expensify/App/issues/10539

**Comment by dhairyasenjaliya** [comment]
@Luke9389 for fixing this https://github.com/Expensify/App/issues/10520 with this Keyboard Spacer PR I have added a solution here feel free to test out https://github.com/Expensify/App/issues/10520#issuecomment-1226079890

**Comment by Luke9389** [comment]
OK I'm gonna close this because it does not fix the critical blocker.

**Pull request closed without merging**